### ISSUE TITLE
Regarding real data collection

### ISSUE THREAD

**Issue by HaodongZheng** [gabrielecaddeo/visuo-tactile-shape-estimation#1]
A issue that logs problems encountered during data collection on the Panda setup at IIT.

**Comment by HaodongZheng** [comment]
Current issue:
It is difficult to get clear indentation from the DIGIT sensor in the manual operating mode.
As the Panda robot arm is not completely stiff and the workbench is not fixed to the table, once my hand let loose, the indentation becomes way less clear. 
The data collection procedure goes as follows: I first use manual mode to drive the Panda to establish contact between the DIGIT sensor and the object in test on the wooden workbench to get a clear indented area on the DIGIT input image. Then I let my hand loose, both the DIGIT and the object remain in contact, but they move a bit away from each other so the change in the RBG image become less prominent than before.

**Comment by gabrielecaddeo** [comment]
Hi @HaodongZheng, can you try to explain better?

**Comment by HaodongZheng** [comment]
> Hi @HaodongZheng, can you try to explain better?

Yes, I updated the explanation.

**Comment by gabrielecaddeo** [comment]
> Then I let my hand loose, both the DIGIT and the object remain in contact, but they move a bit away from each other so the change in the RBG image become less prominent than before.

Yes, this is true. For our data collection, sometiems we had to keep the contact stable with our hands. I believe you can follow 2 paths:
- you can use one hand to keep the contact stable and with the other hand (with the wireless keyboard) you can save the image and the pose
- modify the code to get the pose when the robot is controlled by the software. You could use the joypad that Andrea Rosasco was using and ask him for help

I am sorry but for the purpose I needed for the paper, It was enough controlling by hand

**Comment by HaodongZheng** [comment]
> > Then I let my hand loose, both the DIGIT and the object remain in contact, but they move a bit away from each other so the change in the RBG image become less prominent than before.
> 
> Yes, this is true. For our data collection, sometiems we had to keep the contact stable with our hands. I believe you can follow 2 paths:
> 
>     * you can use one hand to keep the contact stable and with the other hand (with the wireless keyboard) you can save the image and the pose
> 
>     * modify the code to get the pose when the robot is controlled by the software. You could use the joypad that Andrea Rosasco was using and ask him for help
> 
> 
> I am sorry but for the purpose I needed for the paper, It was enough controlling by hand

No worries. I think the second approach makes more sense to me, I will try to ask Andrea when he came back from Florence. I will probably try to make the controller to impose a constant force on the surface. I also want to fix the workbench's relative position to the shelf, is there a recommended way to achieve this without causing damage to the shelf?

**Comment by gabrielecaddeo** [comment]
yes, I believe this is the right way of doing it. You can modify the code in digit-panda by creating a new branch and then push the code
